### PR TITLE
Format code by rustfmt

### PR DESCRIPTION
**Pull request by JohnTitor**
This is a part of #343.
I'll submit PRs little by little because the diff will be too large to review on GitHub.
And at first, we should consider formatting rules as @theotherphil mentioned in https://github.com/image-rs/imageproc/issues/343#issuecomment-512971336.

### CODE REVIEW

File: src/drawing/rect.rs [hunk 1]
```diff
@@ -124,20 +124,22 @@ mod tests {
...
         let mut blended = blue;
         blended.blend(&semi_transparent_red);
 
         let expected = vec![
-            white, white,  white,  white, white,
-            white,  blue,   blue,   blue, white,
-            white,  blue, blended,  blue, white,
-            white,  blue,   blue,   blue, white,
-            white, white,  white,  white, white
+            white, white, white, white, white, white, blue, blue, blue, white, white, blue,
```
Comment (theotherphil) [inline src/drawing/rect.rs]: This should remain formatted as a 5x5 grid.

File: src/filter/median.rs [hunk 2]
```diff
@@ -242,12 +256,15 @@ impl HistSet {
             data.push([0u32; 256]);
         }
 
-        HistSet { data, expected_count }
+        HistSet {
```
Comment (theotherphil) [inline src/filter/median.rs]: What's the rule that's being applied here? This line is already very short. Does rustfmt now enforce one row per field for all struct initialisations?

File: src/filter/mod.rs [hunk 3]
```diff
@@ -692,10 +692,7 @@ mod tests {
 
     #[test]
     fn test_filter3x3_with_results_outside_input_channel_range() {
-        let kernel: Vec<i32> = vec![
-            -1, 0, 1,
-            -2, 0, 2,
-            -1, 0, 1];
+        let kernel: Vec<i32> = vec![-1, 0, 1, -2, 0, 2, -1, 0, 1];
```
Comment (theotherphil) [inline src/filter/mod.rs]: This should remain a 3x3 grid.

File: src/filter/mod.rs [hunk 4]
```diff
@@ -754,16 +751,10 @@ mod tests {
             9, 4;
             8, 1);
 
-        let k: Vec<f32> = vec![
-            0.1, 0.2, 0.1,
-            0.2, 0.4, 0.2,
-            0.1, 0.2, 0.1
-        ];
+        let k: Vec<f32> = vec![0.1, 0.2, 0.1, 0.2, 0.4, 0.2, 0.1, 0.2, 0.1];
```
Comment (theotherphil) [inline src/filter/mod.rs]: And this

File: src/filter/mod.rs [hunk 5]
```diff
@@ -775,10 +766,7 @@ mod tests {
     #[bench]
     fn bench_filter3x3_i32_filter(b: &mut Bencher) {
         let image = gray_bench_image(500, 500);
-        let kernel: Vec<i32> = vec![
-            -1, 0, 1,
-            -2, 0, 2,
-            -1, 0, 1];
+        let kernel: Vec<i32> = vec![-1, 0, 1, -2, 0, 2, -1, 0, 1];
```
Comment (theotherphil) [inline src/filter/mod.rs]: And this

File: tests/regression.rs [hunk 6]
```diff
@@ -136,52 +166,98 @@ fn test_equalize_histogram_grayscale() {
...
 fn test_affine_nearest_rgb() {
     fn affine_nearest(image: &RgbImage) -> RgbImage {
-        let root_two_inv = 1f32/2f32.sqrt()*2.0;
+        let root_two_inv = 1f32 / 2f32.sqrt() * 2.0;
         let hom = Projection::from_matrix([
-            root_two_inv, -root_two_inv,  50.0,
-            root_two_inv,  root_two_inv, -70.0,
-            0.0         , 0.0          , 1.0,
-        ]).unwrap();
+            root_two_inv,
```
Comment (theotherphil) [inline tests/regression.rs]: And this

File: tests/regression.rs [hunk 7]
```diff
@@ -136,52 +166,98 @@ fn test_equalize_histogram_grayscale() {
...
 fn test_affine_bilinear_rgb() {
     fn affine_bilinear(image: &RgbImage) -> RgbImage {
-        let root_two_inv = 1f32/2f32.sqrt()*2.0;
+        let root_two_inv = 1f32 / 2f32.sqrt() * 2.0;
         let hom = Projection::from_matrix([
-            root_two_inv, -root_two_inv,  50.0,
-            root_two_inv,  root_two_inv, -70.0,
-            0.0         , 0.0          , 1.0,
-        ]).unwrap();
+            root_two_inv,
```
Comment (theotherphil) [inline tests/regression.rs]: And this

File: src/filter/median.rs (hunk 2 above)
Comment (JohnTitor) [reply src/filter/median.rs]: Uh, indeed, it's aggressive a bit. I think the reason is here https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#struct. I added `#[rustfmt::skip]`

**Comment by JohnTitor** [comment]
I added `#[rustfmt::skip]` for grid styles.

File: src/filter/median.rs [hunk 8]
```diff
@@ -242,12 +256,13 @@ impl HistSet {
             data.push([0u32; 256]);
         }
 
+        #[rustfmt::skip]
         HistSet { data, expected_count }
```
Comment (theotherphil) [inline src/filter/median.rs]: Sorry to be awkward, but I don't think it makes sense to opt out of the standard formatting for this one line - I was just a bit surprised that rustfmt decided to split this line.

Please could you revert this line and then I'll merge the PR.

Comment (JohnTitor) [reply src/filter/median.rs]: Okay, I reverted the change.

File: src/filter/median.rs (hunk 2 above)
Comment (theotherphil) [reply src/filter/median.rs]: Thanks for the link. I think we should just go with the default here, even if it's a bit odd.

**Comment by theotherphil** [comment]
Thanks!

**Pull request merged**